uniflow/op/extract/ipynb_op.py

```diff
@@ -0,0 +1,44 @@
+"""Extract txt op."""
+import copy
+import re
+from typing import Sequence
+
+from uniflow.node.node import Node
+from uniflow.op.op import Op
+
+
+class ExtractIpynbOp(Op):
+    """Extract txt Op Class."""
+
+    def __call__(self, nodes: Sequence[Node]) -> Sequence[Node]:
+        """Run Model Op.
+
+        Args:
+            nodes (Sequence[Node]): Nodes to run.
+
+        Returns:
+            Sequence[Node]: Nodes after running.
+        """
+        try:
+            import nbformat  # pylint: disable=import-outside-toplevel
+            from nbconvert import (  # pylint: disable=import-outside-toplevel
+                MarkdownExporter,
+            )
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError(
+                "Please install nbformat and nbconvert to load ipynb file. You can use `pip install nbformat nbconvert` to install them."
+            ) from exc
+        output_nodes = []
+        for node in nodes:
+            value_dict = copy.deepcopy(node.value_dict)
+            nb = nbformat.read(value_dict["filename"], as_version=4)
```

Comment:
qq: what does `as_version` means here? Should we make `as_version` configurable?